app/templates/components/page-footer.hbs

```diff
@@ -97,6 +97,15 @@
             </LinkTo>
           </li>
         {{/if}}
+        <li>
+          <ExternalLinkTo
```

Comment:
Is it intentional to use `ExternalLinkTo` rather than `LinkTo`? The jenkins page is an internal route, so normally it would make more sense to just use `LinkTo`. And then we wouldn't need to define the URL in config. BUT since the jenkins page doesn't show on `org`, perhaps you're using the external link to allow the `org` environment to link to the page. In which case, that makes a lot of sense.

But now I'm thinking if that's the case, a better approach might be to allow the jenkins page to be seen on `org` and use `LinkTo`. Although IIRC, one concern about that was duplicate content causing SEO issues, so maybe not. WDYT?

Reply:
@sorta, happy to go with whatever works best for you - just an old dev in a marketing body trying to make things work!

Reply:
Gotcha :smile: I think we can stick with the external link for now. :+1: